source/development/packaging/generate-dashboard-package.rst

```diff
@@ -20,15 +20,22 @@ Build manually
 
 Requirements:
 
--  Docker
+-  Docker: see `Docker installation guide <https://docs.docker.com/engine/install/>`_
+-  zip: see `zip installation guide <https://www.tecmint.com/install-zip-and-unzip-in-linux/>`_
+-  NVM (Node Version Manager): see `NVM installation guide <https://github.com/nvm-sh/nvm#installing-and-updating>`_
+
+   -  ``nvm install v|WAZUH_DASHBOARD_NODE_VERSION|``: installs Node.js version v|WAZUH_DASHBOARD_NODE_VERSION|
+   -  ``nvm use v|WAZUH_DASHBOARD_NODE_VERSION|``: sets the current Node.js version to v|WAZUH_DASHBOARD_NODE_VERSION|
+
+-  Yarn v1.22.22 (Node Package Manager): see `Yarn installation guide <https://classic.yarnpkg.com/en/docs/install/>`_
```

Comment:
**suggestion**: if we consider defining the exact version of yarn, we could add a variable instead of hardcoding the value.